### ISSUE TITLE
Ember Canary - Property state is undefined on renderNode cause of deprecation

### ISSUE THREAD

**Issue by mattgrannary** [ef4/liquid-fire#362]
I'm getting an error in ember-internals when running ember#canary.

```
TypeError: undefined is not an object (evaluating 'renderNode.state.stream')
```

I'm guessing that the `state` property has been fully removed now following the deprecation warning to start using `_state`. Is there any reason for continuing to use state? Those properties were deprecated sometime in v1.8.0.

Could check if it exists first and fallback to using `_state`.

**Comment by ef4** [comment]
This is a different `state`. It's part of the private streams API.

We need to switch to getState.

**Comment by mattgrannary** [comment]
I see. Ya that worked for getting the right state property and accessing the stream. Getting another error now though. Not sure if it's in relation to this stream.

```
Error: Stream error: compute not implemented
```

Does that ring a bell?

**Comment by ef4** [comment]
Yes. We need to import a different thing from the stream module now. We're
getting BasicStream when we want Stream.

I expect to publish a fix shortly, I just got detoured fixing ember-qunit
too, because that was messing up LF's test suite under canary.

**Comment by mattgrannary** [comment]
No worries. Thanks!

**Comment by mattgrannary** [comment]
Perfect. That did the trick. Changed the require to get `Stream` instead of `default` and all is well. Do you need help getting this patch in?

**Comment by ef4** [comment]
Sure, happy to take a PR, as long as it's also backward-compat.

**Comment by trentmwillis** [comment]
@ef4 any chance we could get a release with this fix in place?

**Comment by ef4** [comment]
Released 0.21.3
